### ISSUE TITLE
Requirements.txt

### ISSUE THREAD

**Issue by hunterowens** [fatlotus/eisenhower#4]

**Comment by fatlotus** [comment]
Er... right now there aren't any. It's that janky.

**Comment by hunterowens** [comment]
That's why its in an issue.

I generally just use the output of pip freeze from my virtualenv to produce
an initial draft.

-Hunter

**Comment by fatlotus** [comment]
No, I mean there literally aren't any requirements. It is stdlib only.

**Comment by hunterowens** [comment]
Impressive. 

—
